go-build/Dockerfile

```
@@ -1,16 +1,16 @@
 FROM hairyhenderson/gomplate:v3.6.0-slim AS gomplate
 FROM docker:19.03.8 AS docker
-FROM docker/compose:1.25.4 AS compose
+FROM docker/compose:1.25.5 AS compose
 FROM vault:0.11.6 AS vault
 FROM hashicorp/terraform:0.12.24 AS terraform
 FROM hashicorp/packer:1.5.5 AS packer
 FROM prom/prometheus:v2.17.1 AS prometheus
 FROM prom/alertmanager:v0.20.0 AS alertmanager
 FROM mikefarah/yq:2.4.2 AS yq
-FROM lachlanevenson/k8s-kubectl:v1.18.0 AS kubectl
+FROM lachlanevenson/k8s-kubectl:v1.18.1 AS kubectl
 FROM lachlanevenson/k8s-helm:v2.16.5 AS helm
 FROM lachlanevenson/k8s-helm:v3.1.2 AS helm3
-FROM golangci/golangci-lint:v1.23.8-alpine AS golangci-lint
+FROM golangci/golangci-lint:v1.24.0-alpine AS golangci-lint
```

Need to pin this due to a bug in golangci-lint:

```suggestion
FROM golangci/golangci-lint:v1.23.8-alpine AS golangci-lint
```